index.js

```diff
@@ -6,23 +6,25 @@ var EventEmitter = require('events').EventEmitter;
 
 function Compare(a, b, options) {
   this.options = options ? options : {};
+  this._horizontal = options.orientation === 'horizontal';
   this._onDown = this._onDown.bind(this);
   this._onMove = this._onMove.bind(this);
   this._onMouseUp = this._onMouseUp.bind(this);
   this._onTouchEnd = this._onTouchEnd.bind(this);
   this._ev = new EventEmitter();
   this._swiper = document.createElement('div');
-  this._swiper.className = 'compare-swiper';
+  this._swiper.className = this._horizontal ? 'swiper-horizontal'  : 'compare-swiper';
```

Comment:
could this be simpler if you used `compare-swiper-vertical` and `compare-swiper-horizontal`?

Reply:
Done